examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -12,9 +12,8 @@
    "metadata": {},
```

Comment:
<p><code>window</code> is a constant. give it a proper name and use CAPITALS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Now, there are two parameters FREQUENCY and N_LAST and they are equal. 
FREQUENCY is responsible for how often evaluate metrics.
N_LAST  is responsible for a selected number of last results for evaluation.